smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
When combining smother files, normalizing the paths at write time is necessary and is what I do. But you're correct that at write time when generating smother files it is not necessary to do anything to path. I just thought that it was nicer to have the option to create a smother file with cleaned paths right off the bat when you want. But the aliasing probably shouldn't happen at this stage. That should only be used for combining files.